### PR TITLE
(maint) - README compatibility and managed modules update

### PR DESCRIPTION
**Pull request by pmcmaw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +129,11 @@ Gemfile:
 
 8) Remove your module from being managed via `modulesync` and you can start making use of `pdksync` going forward! Yay no more manual PR creation.
 
+### Compatibility
+----------
+
+Currently this tool has been ran and tested on OSX and linux. **It does not run on Windows.**
```
Comment (HelenCampbell) [inline README.md]: Slight edit suggestion:
"This tool has been developed and tested on OSX and linux. **It currently does not run on Windows.**"

File: managed_modules.yml [hunk 2]
```diff
@@ -1,4 +1,31 @@
 ---
-- puppetlabs-testing
-- puppetlabs-testing1
-- puppetlabs-testing2
+- puppetlabs-accounts
+- puppetlabs-apache
+- puppetlabs-apt
+- puppetlabs-certregen
```
Comment (HelenCampbell) [inline managed_modules.yml]: This isn't converted - and I'm not sure it's within our control to do so?

File: managed_modules.yml [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
+- puppetlabs-postgresql
+# - puppetlabs-satellite_pe_tools
+- puppetlabs-tagmail
+- puppetlabs-tomcat
+- puppetlabs-translate
+# - puppetlabs-vcsrepo
+- puppetlabs-websphere_application_server
+- puppetlabs-bootstrap
+- puppetlabs-concat
+- puppetlabs-execute
```
Comment (HelenCampbell) [inline managed_modules.yml]: *puppetlabs-exec

Comment (pmcmaw) [reply managed_modules.yml]: FML autocorrect

**Comment by pmcmaw** [comment]
@HelenCampbell feedback has been addressed in the following PR: https://github.com/puppetlabs/pdksync/pull/43/files

**Pull request merged**